app/kudos/test_models.py

```diff
@@ -1,22 +0,0 @@
-# -*- coding: utf-8 -*-
-"""Test the Kudos models.
```

Comment:
@chibie Good catch that this test isn't implemented. Should we make a ticket to implement this test?

Reply:
@zacheryschiller i don't think it's worth the effort to test just models... more focus should be on testing request/response cycle and helper functions